server-phoenix/test/helios_web/controllers/events_api/events_api_controller_test.exs

```diff
@@ -3,148 +3,151 @@ defmodule HeliosWeb.WebHooks.EventsAPIControllerTest do
   use ExUnit.Case
   alias Helios.Event
   alias Helios.Repo
+  alias Helios.User
 
   test "Proper status codes on different request", %{conn: conn} do
+    Repo.insert!(%User{
+      user_name: "rvrx",
+      public_key: File.read!("test/helios_web/controllers/events_api/pub.pem")
+    })
+
+    # All valid except the JSON body itself
     conn_1 =
       conn
       |> put_req_header("content-type", "application/json")
+      |> put_req_header(
+        "http-auth",
+        "aXGMNYY/AetzCz2Gdmkx4VeQj5sWzEJ6cF/n0ThioysMFxnhwLjrkQJKHIjjKM6EuFu0DHQLVyvnHM6wNJylRsdT33yLFyu0+qFeAmFucCtbt7sgXsIUXHAOfKR6eTnxLjouYFW5rj3RNfcebsQZyERfPt6Jsc+A4nSk/w3M1p4PpeMOCU1IASy1eSHurpcYJlaJaRdwO5g7wbD36nygdL2lFIh/wTiOqKTo+Vys0D9a2Hcmg3dr01hNn/8sG/CLJALta3zKW763FMFJ4yJrfNgSwDB/mijl4UBBIHx0ZY0AJq8G+0lUof8gaKqAizYM9EirxH9YLaA4iI8Jo9J+Kg=="
```

Comment:
Could you add a comment at the top of the file for how to re-create these payloads from the CLI (assuming that's how you did it)?